MREUnityRuntime/MREUnityRuntimeLib/Patching/PatchingUtils.cs

```diff
@@ -40,6 +40,26 @@ internal static class PatchingUtilMethods
             }
         }
 
+        public static string GeneratePatch(string _old, string _new)
+        {
+            if (_old == null && _new != null)
+            {
+                return _new;
+            }
+            else if (_new == null)
+            {
+                return null;
+            }
```

Comment:
Couldn't this just be if(_old == null) {return _new;}  - whether _new is null or not doesn't matter, we just return new

Reply:
or rather if(_old == null || _new == null || _old != _new) return _new;?


---
In reply to: [256591338](https://github.com/Microsoft/mixed-reality-extension-unity/pull/28#discussion_r256591338) [](ancestors = 256591338)

Reply:
Yes, but I was following the pattern used elsewhere in the file.

---
In reply to: [256592130](https://github.com/Microsoft/mixed-reality-extension-unity/pull/28#discussion_r256592130) [](ancestors = 256592130,256591338)